src/Sensors/Sensor.h

```diff
@@ -1,8 +1,13 @@
 #ifndef SENSOR_H
 #define SENSOR_H
 
-#include "../RecordData/RecordData.h"
 #include "../Constants.h"
+#include "../Math/CircBuffer.h"
+#include "../BlinkBuzz/BlinkBuzz.h"
+#include <string.h>
+#include <stdio.h>
```

Comment:
What are string.h and stdio.h being used for? I'm pretty sure Arduino.h should provide most of the functions we're using from them.